cloud

```diff
@@ -1243,6 +1243,20 @@ function pubsub_withdraw() {
     pubsub_topic_withdraw "$project" "$pick_notifications_trigger_topic"
 }
 
+function storage_deploy(){
```

Comment:
Please observe the coding conventions regarding placement of the opening brace `{`.

Please document this and the other function, following conventions set by other functions here.

Reply:
Okay